101-hdinsight-hbase-replication-geo/README.md

```diff
@@ -0,0 +1,62 @@
+# Deploy a HDInsight HBase replication across two regions
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-hdinsight-hbase-replication-geo%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-hdinsight-hbase-replication-geo%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template allows you to create a HDInsight HBase replication with two virtual networks in two different regions. After it is done, you need to configure static IP addresses for the Zookeeper nodes before you can enable HBase replication using script action. For more information, see [Configure HBase replication](https://docs.microsoft.com/azure/hdinsight/hdinsight-hbase-replication).
+
+Some of the hard-coded values in the template:
+
+**VNet 1**
```

Comment:
I would consider remove this unless there's a good/necessary reason (i.e. other then just informing) - since there's nothing keeping this in sync with what's really in the template...